packages/business/src/server.js

```diff
@@ -67,7 +67,7 @@ passport.use(
           firstName: profile.name.givenName,
           lastName: profile.name.familyName,
         },
-        { upsert: true, new: true },
+        { upsert: true, new: true, setDefaultsOnInsert: true },
```

Comment:
Why was this flag added?

Reply:
This was because without it, the default values for mongo (namely the 'user' default for role) are not used.